certification/engine/engine.go

```diff
@@ -88,6 +88,21 @@ var nameToChecksMap = map[string]certification.Check{
 	validateOperatorBundle.Name():         validateOperatorBundle,
 }
 
+var containerPolicyChecks = map[string]certification.Check{
```

Comment:
We can get rid of nameToCheckMap initialization (line 76). This var should be initialized with either `containerPolicyChecks` or `operatorPolicyChecks` based on the flag, set by the consumer.

Reply:
Agreed.  Was planning to do that in a future PR.